app/components/user-profile-bio.js

```diff
@@ -204,6 +226,7 @@ export default Component.extend(ValidationErrorDisplay, Validations, {
       const firstName = this.firstName;
       const lastName = this.lastName;
       const email = this.email;
+      const preferredEmail = this.preferredEmail;
```

Comment:
I don't think this property will every by part of the sync. I'd have to look further, but I don't think we return a full user record from the sync run, it's an object based on what is in LDAP.

Reply:
thanks! taking it out.